### DISCUSSION TITLE
Google Batch: How to selectively disable spot instance for a process

### DISCUSSION THREAD

**Discussion by VasLem** [nextflow-io/nextflow#5133] [Q&A] [answered]
In `nextflow.config` it is possible to activate the usage of spot instances by setting `google.batch.spot = True` . I have a long running process that I don't want to get preempted. More specifically, I keep getting the error 50002 (which remains unhandled by Nextflow) and want to assess if this error has anything to do with the instance being a spot one. I would therefore like to use a normal instance for that process. Thanks in advance for any pointers!

**Comment by vinisalazar** [comment]
I'm getting a similar problem

**Comment by vinisalazar** [comment]
I would just like to add that I get this problem even with the standard provisioning model.

**Comment by bentsherman** [comment, marked as answer]
It is not possible to set the spot flag for specific processes. But if your main issue is the 50002 error, we will address it in #5141
